### DISCUSSION TITLE
android11 使用PAGImageView 打开了硬件加速，还是出现崩溃

### DISCUSSION THREAD

**Discussion by EmiliaSagiri** [Tencent/libpag#2318] [Q&A] [answered]
出错日志：
01-01 08:05:53.331 13889 13889 W System.err: java.lang.IllegalArgumentException: Software rendering doesn't support hardware bitmaps
01-01 08:05:53.332 13889 13889 W System.err:    at android.graphics.BaseCanvas.onHwBitmapInSwMode(BaseCanvas.java:632)
01-01 08:05:53.332 13889 13889 W System.err:    at android.graphics.BaseCanvas.throwIfHwBitmapInSwMode(BaseCanvas.java:639)
01-01 08:05:53.332 13889 13889 W System.err:    at android.graphics.BaseCanvas.throwIfCannotDraw(BaseCanvas.java:73)
01-01 08:05:53.332 13889 13889 W System.err:    at android.graphics.BaseCanvas.drawBitmap(BaseCanvas.java:113)
01-01 08:05:53.332 13889 13889 W System.err:    at android.graphics.Canvas.drawBitmap(Canvas.java:1600)
01-01 08:05:53.332 13889 13889 W System.err:    at org.libpag.PAGImageView.onDraw(SourceFile:9)
01-01 08:05:53.332 13889 13889 W System.err:    at android.view.View.draw(View.java:22352)
01-01 08:05:53.332 13889 13889 W System.err:    at android.view.View.draw(View.java:22225)
01-01 08:05:53.332 13889 13889 W System.err:    at android.view.ViewGroup.drawChild(ViewGroup.java:4516)
01-01 08:05:53.332 13889 13889 W System.err:    at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4277)
01-01 08:05:53.332 13889 13889 W System.err:    at android.view.View.draw(View.java:22223)

libpag版本：4.3.57
android版本：android11
硬件芯片：高通8155
![image](https://github.com/Tencent/libpag/assets/87418366/38e3648f-6475-4980-8cbb-aed83008aedf)
已打开硬件加速，想问下为啥还是报错

**Comment by domchen** [comment]
看看是不是自己把手机开发者模式里的硬件加速渲染开关给关了。

**Reply by EmiliaSagiri** [reply, marked as answer]
实际原因是 代码里面设置了eightbitlab.com.blurview这个库的参数，setBlurRadius(4f)，把这行去掉，报错就没了
